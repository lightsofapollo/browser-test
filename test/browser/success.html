<!DOCTYPE HTML>
<html>
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <title>body</title>
</head>
<body>
<script src="/socket.io/socket.io.js"></script>
<script src="/build/browser_test.js" type="text/javascript" charset="utf-8"></script>
<script type="text/javascript" charset="utf-8">
browserTest.createSuite(2, 'success.html');

browserTest.createTest('start one', new Date());
browserTest.passTest();

browserTest.createTest('start two', new Date());
browserTest.passTest();

browserTest.finishSuite(new Date());
</script>
</body>
</html>
