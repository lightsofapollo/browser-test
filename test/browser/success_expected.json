{
  "ok": true,
  "asserts": [
    {
      "ok": true,
      "number": 1,
      "name": "start one"
    },
    {
      "ok": true,
      "number": 2,
      "name": "start two"
    }
  ],
  "pass": [
    {
      "ok": true,
      "number": 1,
      "name": "start one"
    },
    {
      "ok": true,
      "number": 2,
      "name": "start two"
    }
  ],
  "fail": [],
  "todo": [],
  "errors": [],
  "plan": {
    "start": 1,
    "end": 2
  }
}